### DISCUSSION TITLE
Open Webui: minimal configuration

### DISCUSSION THREAD

**Discussion by flefevre** [open-webui/open-webui#1850] [Q&A]
Dear openwebui team,
i did'nt find in the documentation the minimal hardware ressource to run OpenWebui in term of CPU and Memory.
Could you give me the link or the information?

Thanks
François

**Comment by justinh-rahb** [comment]
For a good time, you want probably at least 2 vCPU cores and at least 2GB of RAM allocated if it were running on top of a VM. I go with double that. Disk space is another issue, the Docker image itself is between 3.5GB and 8GB depending on the version, and you'd want some room for downloading updates so I'd say you should expect to need at least 15-20GB, and that's before considering uploaded documents for RAG and eventual growth of your database (which you could run externally now since v0.1.122).
